tasks/eva_2380/find_FP_variants.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+from argparse import ArgumentParser
+from collections import defaultdict
+
+import pysam
+
+
+def fetch_bases(fasta, contig, start, length):
+    """
+    Returns a subsection from a specified FASTA contig. The start coordinate is 1-based.
+    """
+    zero_base_start = start - 1
+    end = zero_base_start + length
+    new_ref = fasta.fetch(reference=contig, start=zero_base_start, end=end)
+    return new_ref
+
+
+def assessment_result_per_variants_from_vcf(vcf_file):
+    # Two samples in this VCF first the truth set then the query set
+    print('Read Assessment VCF: ' + vcf_file, file=sys.stderr)
+    assessment_result_per_variants = {}
+    with pysam.VariantFile(vcf_file, 'r') as vcf_in:
+        for vcf_record in vcf_in:
+            try:
+                if vcf_record.samples['QUERY']['BD'] in ['FP']:
```

Comment:
```suggestion
                if vcf_record.samples['QUERY']['BD'] == 'FP':
```
(unless you want to generalise it in the future)